include/cantera/zeroD/ReactorNet.h

```diff
@@ -141,6 +143,46 @@ class ReactorNet : public FuncEval
     //! (time or space). Returns the new value of the independent variable [s or m].
     double step();
 
+    //! Solve directly for the steady-state solution.
+    //!
+    //! This approach is generally more efficient than time marching to the
+    //! steady-state, but imposes a few limitations:
+    //!
+    //! - The volume of control volume reactor types (such as Reactor and
+    //!   IdealGasMoleReactor) must be constant; no moving walls can be used.
+    //! - The mass of constant pressure reactor types (such as ConstPressureReactor and
+    //!   IdealGasConstPressureReactor) must be constant; if flow devices are used,
+    //!   inlet and outlet flows must be balanced.
+    //! - The solver is currently not compatible with the ConstPressureMoleReactor or
+    //!   IdealGasConstPressureMoleReactor classes.
+    //! - Only "ideal gas" reactor types can be used for when the energy equation is
```

Comment:
I don't think ideal gas needs to be in quotes here.
```suggestion
    //! - Only ideal gas reactor types can be used when the energy equation is
```